src/engine/Operation.cpp

```diff
@@ -284,7 +287,7 @@ void Operation::updateRuntimeInformationWhenOptimizedOut(
   auto setStatus = [&status](RuntimeInformation& rti,
                              const auto& self) -> void {
     rti.status_ = status;
-    rti.totalTime_ = 0;
+    rti.totalTime_ = millis::zero();
```

Comment:
```suggestion
    rti.totalTime_ = 0ms;
```
(you need a using namespace std::chrono_literals or whatever it is called)
(also use consistently at the other places).